NBitcoin.Tests/NBitcoin.Tests.csproj

```diff
@@ -161,6 +161,9 @@
     <Content Include="data\block169482.txt">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="data\testnet-20.csv">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>

```

Comment:
I'd rather hardcode it. What do you think @NicolasDorier ?

Reply:
I've given the file a better name.